python/ray/tests/test_metrics_agent.py

```diff
@@ -581,13 +579,16 @@ def verify_components():
             components = set()
             for sample in samples:
                 components.add(sample.labels["Component"])
+
+        # NOTE: when Actor.__init__ runs slow, it might also show up
+        # in the metrics.
```

Comment:
Adding a simple "time.sleep(5)" could repro this.

Reply:
why don't we just wait for 

```
ray.get(a.__ray_ready__.remote())
```
?

Reply:
so we can keep more strict assert here